components/ProposedGoalsList/index.js

```diff
@@ -0,0 +1,33 @@
+import ProposedGoalPreview from "../ProposedGoalPreview/index.js";
+import styled from "styled-components";
+
+const StyledList = styled.div`
+  display: grid;
+  grid-template-columns: repeat(2, 1fr);
+  gap: 15px;
+  padding-bottom: 25px;
+`;
+
+const StyledCard = styled.div`
+  padding: 20px;
+  border-radius: 15px;
+  background-color: ${(props) => props.backgroundColor};
+  box-shadow: 0 2px 6px rgb(109 94 255 / 10%);
+`;
```

Comment:
There is most certainly a semantic HTML element that could fit your cards. 😬

Reply:
I replaced the div element with an `article` element.